lib/pxconfig.js

```diff
@@ -58,7 +58,7 @@ PX_DEFAULT.API_TIMEOUT_MS = 3000;
 PX_DEFAULT.BLOCKING_SCORE = 70;
 PX_DEFAULT.COOKIE_SECRET_KEY = 'cookie_secret_key';
 PX_DEFAULT.AUTH_TOKEN = 'auth_token';
-PX_DEFAULT.IP_HEADER = 'px-user-ip';
+PX_DEFAULT.IP_HEADERS = ['px-user-ip'];
```

Comment:
As you can see, I kept the existing setting. If in all other enforcers it's `X-Forwarded-For`, then I'll change it here as well.